src/ds3.c

```diff
@@ -428,6 +432,19 @@ void ds3_request_set_metadata(ds3_request* _request, const char* name, const cha
     g_free(prefixed_name);
 }
 
+void ds3_request_set_byte_range(ds3_request* _request, int64_t rangeStart, int64_t rangeEnd) {
+    char* range_value;
+    range_value = g_strdup_printf("bytes=%ld-%ld", rangeStart, rangeEnd);
```

Comment:
Just looked it up. Our API says it works like the Amazon one does, and the _Amazon_ API only accepts one range. So ours might accept more. How would multiple ranges work with request length and offset?